cmd/podman/main.go

```diff
@@ -67,8 +67,10 @@ var mainCommands = []*cobra.Command{
 	systemCommand.Command,
 }
 
+var cmdUsage = "podman"
+
 var rootCmd = &cobra.Command{
-	Use:  "podman",
+	Use:  cmdUsage,
```

Comment:
Why not set this to 
path.Base(os.Args[0])

Then you don't need to change anything else.

Reply:
Good idea, done.

Reply:
path.Base(os.Executable()) might be cleaner.